##RAM_MAP
uint8  ID
uint8  AckPolicy
uint8  AlarmLEDPolicy
uint8  TorquePolicy
uint8  Reserved2
uint8  MaxTemperature
uint8  MinVoltage
uint8  MaxVoltage
uint8  AccelerationRatio
uint8  MaxAccelerationTime
uint8  DeadZone
uint8  SaturatorOffset
uint16 SaturatorSlope
int8   PWMOffset
uint8  MinPWM
uint16 MaxPWM
uint16 OverloadPWMThreshold
uint16 MinPosition
uint16 MaxPosition
uint16 PositionKp
uint16 PositionKd
uint16 PositionKi
uint16 PositionFeedforward1stGain
uint16 PositionFeedforward2ndGain
uint16 VelocityKd
uint16 VelocityKi
uint8  LEDBlinkPeriod
uint8  ADCFaultCheckPeriod
uint8  PacketGarbageCheckPeriod
uint8  StopDetectionPeriod
uint8  OverloadDetectionPeriod
uint8  StopThreshold
uint8  InpositionMargin
uint8  Turn
int8   CalibrationDifference_L
int8   CalibrationDifference_H
uint8  StatusError
uint8  StatusDetail
uint8  Reserved7
uint8  Reserved8
uint8  TorqueControl
uint8  LEDControl
uint8  Voltage
uint8  Temperature
uint8  CurrentControlMode
uint8  Tick
uint16 CalibratedPosition
uint16 AbsolutePosition
int16  DifferentialPosition
uint16 PWM
uint16 Reserved9
uint16 AbsoluteGoalPosition
uint16 AbsoluteDesiredTrajectoryPosition
int16  DesiredVelocity
